src/packages/core/components/icon/icon.element.ts

```diff
@@ -16,29 +16,48 @@ export class UmbIconElement extends UmbLitElement {
 	@state()
 	private _color?: string;
 
+	@state()
+	private _fallbackColor?: string;
+
+	/**
+	 * Color alias or a color code directly.\
+	 * If a color has been set via the name property, this property will override it.
+	 * */
 	@property({ type: String })
 	public set color(value: string) {
-		if (!value) return;
-		this.#setColorStyle(value);
+		if (value) {
+			this.#setColorStyle(value, true);
+		} else {
+			this._color = undefined;
+		}
 	}
 	public get color(): string {
 		return this._color ?? '';
 	}
 
-	#setColorStyle(value: string) {
+	#setColorStyle(value: string, dominant = false) {
 		const alias = value.replace('color-', '');
 		const variable = extractUmbColorVariable(alias);
-		this._color = alias ? (variable ? `--uui-icon-color: var(${variable})` : `--uui-icon-color: ${alias}`) : undefined;
+		const color = alias ? (variable ? `--uui-icon-color: var(${variable})` : `--uui-icon-color: ${alias}`) : undefined;
```

Comment:
I am not completely sure I understand the logic here, if it finds a variable it uses that = great. If it doesn't find a variable, it tries to set the icon alias in the `--uui-icon-color` variable. Could you explain that a little bit?

Reply:
This would probably be less confusing with better naming. Basically, it doesn't necessary have to be an alias, so a variable name of "color" may make more sense.
This means that `icon.element` allows any color, not just a color alias. It just checks if the color is an alias for an existing color variable. Hope that makes sense.

This PR really shouldnt have been so much about the icon.element 😅 but I'll see about adding these changes.